.github/workflows/amd64_linux_cmake_java.yml

```diff
@@ -32,8 +32,9 @@ jobs:
           sudo apt update
           sudo apt install -y ninja-build ccache swig
           swig -version
+          echo "JAVA_HOME=$JAVA_HOME_21_X64" >> $GITHUB_ENV
       - name: Check java
-        run: java -version
+        run: $JAVA_HOME/bin/java -version
```

Comment:
you could also have used previously:
```yml
echo "$JAVA_HOME/bin" >> "$GITHUB_PATH"
```
ref: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#adding-a-system-path

note: thus you should have `java`, `javac` and `jar` in sync...

Reply:
Fixed in [5e4007f](https://github.com/google/or-tools/pull/4966/commits/5e4007fcddbf5613538a8de7d0ae13df31d7e5dc)